src/box/sql/build.c

```diff
@@ -145,13 +145,14 @@ sql_space_column_is_in_pk(struct space *space, uint32_t column)
 int
 sqlCheckIdentifierName(Parse *pParse, char *zName)
 {
-	ssize_t len = strlen(zName);
+	size_t len = strlen(zName);
 	if (len > BOX_NAME_MAX) {
 		diag_set(ClientError, ER_IDENTIFIER,
 			 tt_cstr(zName, BOX_INVALID_NAME_MAX));
 		pParse->is_aborted = true;
 		return -1;
 	}
+	assert(len < (size_t)INT32_MAX);
```

Comment:
If `zName` is expected to fit in int, you can just mark the SVACE issue as false-positive. This particular change doesn't make the code any better, because you still pass `size_t` for `int` to `identifier_check`.

Reply:
Relevant SVACE issue is closed as "Won't fix".